python/pyarrow/_hdfs.pyx

```diff
@@ -93,9 +93,10 @@ cdef class HadoopFileSystem(FileSystem):
         Instantiate HadoopFileSystem object from an URI string.
 
         The following two calls are equivalent
-        * HadoopFileSystem.from_uri('hdfs://localhost:8020/?user=test'
-                                    '&replication=1')
-        * HadoopFileSystem('localhost', port=8020, user='test', replication=1)
+
+        * ``HadoopFileSystem.from_uri('hdfs://localhost:8020/?user=test&replication=1')`` # noqa: E501
+
+        * ``HadoopFileSystem('localhost', port=8020, user='test', replication=1)`` # noqa: E501
```

Comment:
```suggestion
        * ``HadoopFileSystem.from_uri('hdfs://localhost:8020/?user=test\
&replication=1')``
        * ``HadoopFileSystem('localhost', port=8020, user='test', \
replication=1)``
```

It's a bit ugly in the source code, but this AFAIK the best way to deal with this issue. In a console checking the docstring this will look like a single line.